eldritch/encounters.py

```diff
@@ -144,11 +144,101 @@ def Society4(char):
+def Administration6(char):
+  # Administration 6 is identical to Administration 1
+  return Administration1(char)
 def Administration7(char):
   check = events.Check(char, "will", -2)
   gain = events.Gain(char, {"dollars": 8})
   arrest = events.Arrested(char)
   return events.PassFail(char, check, gain, arrest)
+
+def Library1(char):
+  check = events.Check(char, "will", -1)
+  tome = events.Draw(char, "unique", 1) # TODO: this is actually draw the first tome
+  move = events.ForceMovement(char, "University")
+  return events.PassFail(char, check, tome, move)
+def Library2(char):
+  check = events.Check(char, "will", 0)
+  move = events.ForceMovement(char, "University")
+  spell = events.Draw(char, "spells", 2)
+  unique = events.Draw(char, "unique", 1)
+  cond = events.Conditional(char, check, "successes", {0: move, 1: spell, 2: unique})
+  return events.Sequence([check, cond], char)
+def Library3(char):
+  check = events.Check(char, "lore", -2)
+  die = events.DiceRoll(char, 1)
+  gain = events.Gain(char, {"clues": die})
+  loss = events.Loss(char, {"stamina": 2, "sanity": 2})
+  return events.PassFail(char, check, events.Sequence([die, gain]), loss)
+def Library4(char):
+  return events.Nothing() # TODO: GO TO DREAMLANDS!! DO NOT PASS GO! but do return here
+def Library5(char):
+  return events.Loss(char, {"sanity": 1})
+def Library6(char):
+  prereq = events.AttributePrerequisite(char, "dollars", 4, "at least")
+  pay = events.Loss(char, {"dollars": 4})
+  move = events.ForceMovement(char, "University")
+  return events.PassFail(char, prereq, pay, move)
+def Library7(char):
+  check = events.Check(char, "luck", -2)
+  money = events.Gain(char, {"dollars": 5})
+  return events.PassFail(char, check, money, events.Nothing())
+
+def Science1(char):
+  return events.Bless(char)
+def Science2(char):
+  spell = events.Draw(char, "spells", 1)
+  check = events.Check(char, "fight", -1)
+  lose = events.Nothing() # TODO: lose an item of your choice
+  fight = events.PassFail(char, check, events.Nothing(), lose)
+  return events.Sequence([spell, fight], char) 
+def Science3(char):
+  prereq = events.AttributePrerequisite(char, "dollars", 2, "less than") # TODO: the money is a lie, it's actually number of spells
+  unique = events.Draw(char, "unique", 1)
+  move = events.ForceMovement(char, "University")
+  return events.PassFail(char, prereq, events.Sequence([unique, move], char), events.Nothing())
+def Science4(char):
+  stamina = events.Loss(char, {"stamina": 2})
+  ally = events.DrawSpecific(char, "allies", "Arm Wrestler") # TODO: prereq on the ally being in the deck
+  return events.BinaryChoice(char, "Arm Wrestle?", "Yes", "No", events.Sequence([stamina, ally], char), events.Nothing())
+def Science5(char):
+  check = events.Check(char, "luck", 0)
+  die = events.DiceRoll(char, 1)
+  win = events.SplitGain(char, "stamina", "sanity", die)
+  loss = events.Gain(char, {"stamina": 1})
```

Comment:
While this is correct, I had to do a double-take because the stamina gain was named loss. Can you name it something else?

Reply:
changed to coffee